.github/workflows/lint-checker.yml

```diff
@@ -47,4 +47,4 @@ jobs:
         with:
           node-version: '12.x'
       - run: npm install -g markdownlint-cli@0.25.0
-      - run: markdownlint '**/*.md' --ignore node_modules
+      - run: markdownlint '**/*.md' --ignore node_modules --ignore e2e/charts/nacos/README.md
```

Comment:
Why ignore this document?

Reply:
This readme doesn't belongs to us. It's nacos chart.